runner/utils.go

```diff
@@ -25,12 +25,14 @@ func isTmpDir(path string) bool {
 
 func isIgnoredFolder(path string) bool {
 	paths := strings.Split(path, "/")
-	if len(paths) <= 0 {
+	pathsOnWindows := strings.Split(path, "\\")
```

Comment:
So I think that solution is good :) It does not affect performance, if that is what it worries you